dartagnan/src/main/java/com/dat3m/dartagnan/wmm/axiom/ForceEncodeAxiom.java

```diff
@@ -0,0 +1,30 @@
+package com.dat3m.dartagnan.wmm.axiom;
+
+import com.dat3m.dartagnan.wmm.relation.Relation;
+import com.dat3m.dartagnan.wmm.utils.TupleSet;
+import org.sosy_lab.java_smt.api.BooleanFormula;
+import org.sosy_lab.java_smt.api.SolverContext;
+
+/*
+    This is a fake axiom that forces a relation to get encoded!
+ */
+public class ForceEncodeAxiom extends Axiom {
```

Comment:
Yes. We cannot really encode relations that are not part of any axiom, because the `encodeSets` will be empty.
Having a dummy axiom will trigger our static analyses properly.